src/openms/include/OpenMS/KERNEL/ConsensusMap.h

```diff
@@ -174,7 +177,7 @@ namespace OpenMS
     OPENMS_DLLAPI ConsensusMap(const ConsensusMap& source);
 
     /// Destructor
-    OPENMS_DLLAPI ~ConsensusMap() override;
+    OPENMS_DLLAPI ~ConsensusMap() override = default;
```

Comment:
hmm @cbielow do you know why this doesn't work after defaulting?

Reply:
on mac and linux

Reply:
Try putting it in the CPP: https://stackoverflow.com/questions/3065154/undefined-reference-to-vtable